data/126/021/452/3/1260214523.geojson

```diff
@@ -29,13 +30,14 @@
     "lbl:min_zoom":13.5,
     "mz:hierarchy_label":1,
     "mz:is_current":-1,
+    "mz:is_funky":1,
```

Comment:
We use `mz:is_funky` to indicate there is some sort of problem with a WOF record that makes it suspect. Downsteadm services then either hide that data or give it less weight. 

Probably you meant `mz:is_current` to indicate that this place is a "going concern" (people live there, in the named area).